Formula/jackett.rb

```diff
@@ -0,0 +1,52 @@
+class Jackett < Formula
+  desc "API Support for your favorite torrent trackers"
+  homepage "https://github.com/Jackett/Jackett"
+  url "https://github.com/Jackett/Jackett/archive/refs/tags/v0.20.805.tar.gz"
+  sha256 "5b37a01517d2230879fdd0b5f5a3344abcc7718adee9584c2b7b40e5b99b721b"
+  license "GPL-2.0-only"
+  head "https://github.com/Jackett/Jackett.git", branch: "master"
+
+  depends_on "dotnet"
+  depends_on macos: :high_sierra
+
+  def install
+    cd "src" do
+      os = OS.mac? ? "osx" : OS.kernel_name.downcase
+      arch = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch.to_s
+
+      args = %W[
+        --configuration Release
+        --framework net#{Formula["dotnet"].version.major_minor}
+        --output #{libexec}
+        --runtime #{os}-#{arch}
+        --no-self-contained
+        /p:AssemblyVersion=#{version}
+        /p:FileVersion=#{version}
+        /p:InformationalVersion=#{version}
+        /p:Version=#{version}
+      ]
+
+      system "dotnet", "publish", "Jackett.Server", *args
+    end
+
+    (bin/"jackett").write_env_script libexec/"jackett", DOTNET_ROOT: "${DOTNET_ROOT:-#{Formula["dotnet"].opt_libexec}}"
+  end
+
+  def caveats
+    <<~EOS
+      Configuration files: ~/.config/Jackett (or XDG_CONFIG_HOME environment variable, if set)
+    EOS
+  end
```

Comment:
Unless this is specific to Homebrew it should not be a caveat

Reply:
Okay. I was wondering whether to leave this as a per-user service or make it into a system service. It depends on that choice.

Reply:
Whatever you choose, if it's an upstream default it should not be a caveat.

Reply:
Sure, I'm just saying that if I change it to a system service, then it would not be an upstream default... but let's leave it how it is and remove the caveat.